numba_cuda/numba/cuda/cudadrv/nvvm.py

```diff
@@ -369,37 +369,40 @@ def get_log(self):
     (9, 0),
     (10, 0),
     (10, 1),
+    (10, 3),
```

Comment:
So 10.3 is only available on CUDA 12.9 (not 12.8). As a result it appears the CUDA 12.8 tests failed [on CI](  https://github.com/NVIDIA/numba-cuda/actions/runs/15431269306/job/43429522715?pr=273#step:9:2712 ):

```python
libnvvm : error: -arch=compute_103 is an unsupported option
NVVM_ERROR_INVALID_OPTION
```

So we need to exclude when 10.3 is used